linetools/spectra/xspectrum1d.py

```diff
@@ -855,6 +855,10 @@ def box_smooth(self, nbox, preserve=True, **kwargs):
                     self.sig[orig_pix], new_npix) / np.sqrt(nbox)
             else:
                 new_sig = None
+            if self.co_is_set:
+                new_co = ltu.scipy_rebin(self.co[orig_pix], new_npix)
+            else:
+                new_co = None
 
```

Comment:
Don't know or remember why this would have crept in either....but it's important of course!